app/webpack/stylesheets/cccd-measurements.scss

```diff
@@ -1,2 +1,8 @@
 // CCCD measurements
-$gutter-one-sixth: $gutter / 6;
+
+$gutter-one-sixth: $govuk-gutter / 6;
+
+$gutter-one-quarter: $govuk-gutter / 4;
+$gutter-one-third: $govuk-gutter / 3;
+$gutter-half: $govuk-gutter / 2;
+$gutter-two-thirds: $govuk-gutter - $gutter-one-third;
```

Comment:
These variables come from `govuk_frontend_toolkit` so ideally they should be removed. This is possibly beyond the scope of this work. Could a new ticket be written?

Reply:
Ah $gutter comes from govuk_frontend_toolkit but $govuk-gutter comes from govuk-frontend npm package.

It's just that the proportions like *-one-quarter and *-two-thirds doesn't exist there.

I'll look to create a ticket to remove cccd-measurements.scss 🤞🏾